.github/workflows/pr.yaml

```diff
@@ -12,4 +12,5 @@ jobs:
     - uses: actions/checkout@v4
     - uses: hashicorp/setup-terraform@v3.0.0
     - uses: actions/setup-python@v5.0.0
+    - uses: terraform-linters/setup-tflint@v4
```

Comment:
Is this necessary? I would expect `pre-commit` would do any required setup

Reply:
Hmm, I'll check. I thought it might be required for https://github.com/terraform-linters/setup-tflint?tab=readme-ov-file#checks

Reply:
> Hmm, I'll check. I thought it might be required for https://github.com/terraform-linters/setup-tflint?tab=readme-ov-file#checks

Ah yeah, the pre-commit repo mentions you need to install the deps yourself: https://github.com/antonbabenko/pre-commit-terraform?tab=readme-ov-file#how-to-install